src/features/onboarding/register/StepFee.tsx

```diff
@@ -95,14 +95,15 @@ const StepFee = (props: TStepFeeProps): JSX.Element => {
                 <div className='mr-3 overflow-hidden overflow-ellipsis h-5'>
                   {props.exchangeAddress}
                 </div>
-                <Clipboard
-                  size={20}
-                  className={cn(
-                    'cursor-pointer transition-transform duration-100 ease-in-out text-gray-88',
-                    copying ? 'transform scale-150' : '',
-                  )}
-                  onClick={() => toClipboard()}
-                />
+                <span onClick={() => toClipboard()}>
```

Comment:
Should be a button instead of span

By default <button> is submitting form in case if it is inside of a form, so please use `<button type='button'` and it will behave correctly, and users will enjoy faster navigation

Please update to this:
```jsx
<button type='button' onClick={toClipboard}>
```

Reply:
fixed